src/main/kotlin/io/gitlab/arturbosch/detekt/sonar/rules/DetektRulesDefinition.kt

```diff
@@ -6,6 +6,7 @@ import io.gitlab.arturbosch.detekt.sonar.foundation.REPOSITORY_KEY
 import io.gitlab.arturbosch.detekt.sonar.foundation.LANGUAGE_KEY
 import org.sonar.api.rule.RuleStatus
 import org.sonar.api.server.rule.RulesDefinition
+import java.util.*
```

Comment:
We try to avoid wildcard imports. For consistency reasons may I ask to remove the wildcard import?